mautic/mautic
# Allow save image to contact profile from form

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

This PR allow upload image directly to contact profile from forms.

![image](https://user-images.githubusercontent.com/462477/65511321-37fa5b80-ded7-11e9-99a2-4a45fd49b185.png)


#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create form with file field
3. See properties tab and enable option "Set as profile image"
4. Submit form
5. See If image uploaded by form was set to contact profile.
